config/locales/fr.yml

```diff
@@ -626,12 +660,15 @@ fr:
         shared:
           info_question:
             created_at: Date de création
+            first_interacted_at: First interacted at
             info_question_bottom:
               created_at: Date de soumission
+              first_interacted_at: First interacted at
               published_at: Date de publication
               question_votes_count: Décompte des votes pour la question
             info_question_bottom_full:
               created_at: Date de soumission
+              first_interacted_at: First interacted at
```

Comment:
🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...